wikimedia/wikipedia-ios
# [YiR] Update feature announcement

**Phabricator:** https://phabricator.wikimedia.org/T378507

### Notes
* Add optional background image to `WMFFeatureAnnouncementView`
* Add background asset for YiR announcement
*
### Test Steps
1. Set your device to a valid region (Mexico or Italy)
2. Fresh install the app and login
3. Set the language to Italian or Spanish or English
4. Go through articles and generate 6+ article views in the History tab.
5. Go to developer settings and toggle on Year in Review feature flag.
6. Run the app and background it on the explore feed
7. Open the app again. You should see the modal. 
8. Fresh install the app again, repeat the previous steps
9. Go to an article -  if possible on iPad this time
10. You should see the modal
